appeals/api/src/database/seed/data-test.js

```diff
@@ -654,6 +654,8 @@ export async function seedTestData(databaseConnector) {
 					status = 'invalid';
 				}
 
+				const source = Math.random() < 0.5 ? 'back_office' : 'front_office';
```

Comment:
I would choose: 
- 'citizen' (DEFAULT) -> easier for broadcasting, we're using this generally
- 'back-office-appeals' -> same already defined here: https://github.com/Planning-Inspectorate/appeals-back-office/blob/f3fec0ddf93587cfff91ca566d1d49dac7618a07/packages/appeals/constants/common.js#L1

Reply:
Ah ok so citizen instead of front_office, back-office-appeals instead of back_office?